src/openvino.cc

```diff
@@ -632,12 +639,170 @@ ModelState::ValidateOutputs()
 TRITONSERVER_Error*
 ModelState::AutoCompleteConfig()
 {
-  // TODO: Add the support for auto completing config for this backend.
-  LOG_MESSAGE(
-      TRITONSERVER_LOG_WARN,
-      (std::string("skipping model configuration auto-complete for '") +
-       Name() + "': not supported for openvino backend")
-          .c_str());
+  // Read OV model for autocomplete
+  std::string artifact_name;
+  RETURN_IF_ERROR(
+      ModelConfig().MemberAsString("default_model_filename", &artifact_name));
+  std::string model_path;
+  THROW_IF_BACKEND_INSTANCE_ERROR(ReadModel(artifact_name, &model_path));
+  model_read_ = false;  // Re-read model after autocomplete
```

Comment:
Because this is to re-use the `ModelState::ReadModel` function to obtain the model's inputs and outputs, without having to re-write the `JoinPath()` call and checking file existance before the `ov_core_.read_model`.
The actual read includes parsing the parameters from the config file (if specified) before reading the model, which is not done for the autocomplete read.
It is just easier to simply re-read the model to account for this difference, without having major logic change. I can refactor this part if the re-read is disturbing?

Reply:
No. I think this is fine.